ci/kokoro/docker/build-in-docker-quickstart-cmake.sh

```diff
@@ -63,7 +63,12 @@ else
 fi
 
 "${vcpkg_bin}" remove --outdated --recurse
-"${vcpkg_bin}" install google-cloud-cpp
+if ! "${vcpkg_bin}" install google-cloud-cpp; then
+  # We have seen download errors here, so make a single retry after a
+  # short pause. See https://github.com/microsoft/vcpkg/issues/11073.
+  sleep 5
+  "${vcpkg_bin}" install google-cloud-cpp
```

Comment:
We have https://github.com/googleapis/google-cloud-cpp/blob/master/ci/retry-command.sh that we use in a few places. Could we use that here too?

Reply:
Done.  I also tweaked `retry-command.sh` to act more like it advertises.  PTAL.